statsmodels/robust/norms.py

```diff
@@ -233,7 +233,7 @@ def psi(self, z):
         return test * z + (1 - test) * self.t * np.sign(z)
 
     def weights(self, z):
-        """
+        r"""
```

Comment:
I think you need to remove the escapes below now that you added r.  \| will render \| not |.